src/wallet/wallet_rpc_server.cpp

```diff
@@ -1086,7 +1092,7 @@ namespace tools
     {
       uint64_t mixin = m_wallet->adjust_mixin(req.ring_size ? req.ring_size - 1 : 0);
       uint32_t priority = m_wallet->adjust_priority(req.priority);
-      std::vector<wallet2::pending_tx> ptx_vector = m_wallet->create_transactions_2(dsts, mixin, req.unlock_time, priority, extra, req.account_index, req.subaddr_indices);
+      std::vector<wallet2::pending_tx> ptx_vector = m_wallet->create_transactions_2(dsts, mixin, req.unlock_time, priority, extra, req.account_index, req.subaddr_indices, req.subtract_fee_from_outputs);
```

Comment:
For me, it doesn't make as much sense to add this to the split transfer interface since the fees have to be paid out of certain destinations. If those certain destinations are split over multiple transactions, it's unclear which destinations should pay for transactions if a transaction doesn't contain any "subtractable" destinations. Also, this feature can be expanded upon in a future PR for reviewer's sake, and I'd imagine this feature is mainly useful for small transfers (like customers paying for an item with fee included).

Reply:
simplewallet.cpp already permits applying this feature across split transactions, so either that behavior needs to be well defined and supported or completely disabled.

Then monero-wallet-rpc should be consistent. So if we don't support `transfer_split` in monero-wallet-rpc now, this feature should be disabled for split transactions in simplewallet too.

As far as I can tell, the expected behavior with split transactions should be clear, where the fee is split across any specified destinations in each transaction, and the entire call fails if any transaction cannot be created with subtractable destinations, so my preference is to support this.